commands/cli.py

```diff
@@ -0,0 +1,34 @@
+from cleo.application import Application
+
+import commands
+from starlite import Starlite
+
+
+class Cli:
```

Comment:
Should this by `CLI`?

Reply:
It should, you can add a proposal to change it: https://haacked.com/archive/2019/06/03/suggested-changes/

Reply:
TIL.  Thanks!